Face_Recognition/face_detection.py

```diff
@@ -0,0 +1,35 @@
+# Read a video Stream from Camera  ( Frame by frame )
+
+import cv2
+cap = cv2.VideoCapture(0)
+face_cascade = cv2.CascadeClassifier("haarcascade_frontalface_alt.xml")  # classifier
+while True:
+	ret, frame = cap.read()    # we read 1 img 
+	gray_frame = cv2.cvtColor(frame,cv2.COLOR_BGR2GRAY)
+	if ret==False:
+		continue
+# now we read the img
+										# (gray_frame, scaling factor, no of neighbors)
+	faces = face_cascade.detectMultiscale(gray_frame,1.3,5)   # 1.3 means img will shrink by 30% 
+ 	cv2.imshow("Video Frame",frame)
```

Comment:
Check this error. I think you have to remove all the space before cv2 and use one tab
```
Face_Recognition/face_detection.py:14:0: E0001: inconsistent use of tabs and spaces in indentation (<unknown>, line 14) (syntax-error)
```